docs/2.guide/2.directory-structure/1.components.md

```diff
@@ -303,6 +303,10 @@ Server-only components use [`<NuxtIsland>`](/docs/api/components/nuxt-island) un
 Server components (and islands) must have a single root element. (HTML comments are considered elements as well.)
 ::
 
+::alert{type=warning}
+It is not recommanded to nest island within islands as you will have a waterfall of fetch requests.
```

Comment:
I'm worried we might give the wrong impression, as these all take place in the same network request, with additional server components rendered with 'internal' fetches.

```suggestion
Be careful when nesting islands within other islands as each island adds some extra overhead.
```